.Documentation/articles/innoactive-creator/suspending-interactions.md

```diff
@@ -0,0 +1,7 @@
+# Suspending Interactions
+
+You can let your trainees to interact with a [training object](training-scene-object.md) by adding certain properties to it. By default, objects stay interactive throughout the whole training course: a trainee could push a button and flick a switch at any time. The training feels more real, but it can turn chaotic in complex courses and scenes. To prevent that, you can temporarily disable interactions for a property, and restore it only when your trainee needs it.
```

Comment:
```suggestion
You can let your trainees interact with a [training object](training-scene-object.md) by adding certain properties to it. By default, objects stay interactive throughout the whole training course: a trainee could push a button and flick a switch at any time. The training feels more real, but it can turn chaotic in complex courses and scenes. To prevent that, you can temporarily disable interactions for a property, and restore it only when your trainee needs it.
```

Reply:
No "to" is required.

https://ell.stackexchange.com/questions/121717/let-to-do-let-do-to-let-to-do-to-let-do